api/custom_auth/urls.py

```diff
@@ -20,6 +23,7 @@
         CustomAuthTokenLoginWithMFACode.as_view(),
         name="mfa-authtoken-login-code",
     ),
+    path("", include(throttled_user_router.urls)),
     path("", include("djoser.urls")),
```

Comment:
Looking at what we actually use djoser for, I think we _could_ get away with replacing this with just the overridden viewset instead, right? Maybe that's not a good idea just incase anyone is using the additional endpoints in their open source installation but maybe we could add a comment to remove them in the next major release?

Reply:
Yeah, that's a good point. I will add the comment